src/components/views/dialogs/DeactivateAccountDialog.js

```diff
@@ -0,0 +1,139 @@
+/*
+Copyright 2016 OpenMarket Ltd
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React from 'react';
+
+import sdk from '../../../index';
+import MatrixClientPeg from '../../../MatrixClientPeg';
+import Lifecycle from '../../../Lifecycle';
+import Velocity from 'velocity-vector';
+
+export default class DeactivateAccountDialog extends React.Component {
+    constructor(props, context) {
+        super(props, context);
+
+        this._passwordField;
```

Comment:
Does this do anything in a void context? Would prefer to see an explicit assignment, anyway.